install/linux/docker-ee/ubuntu.md

```diff
@@ -173,47 +173,13 @@ from the repository.
     > Ubuntu distribution, such as `xenial`.
     >
 
-    <ul class="nav nav-tabs">
-      <li class="active"><a data-toggle="tab" data-target="#x86_64_repo">x86_64 / amd64</a></li>
-      <li><a data-toggle="tab" data-target="#s390x_repo">IBM Z (s390x)</a></li>
-      <li><a data-toggle="tab" data-target="#ppc64el_repo">IBM Power (ppc64el)</a></li>
-    </ul>
-    <div class="tab-content">
-    <div id="x86_64_repo" class="tab-pane fade in active" markdown="1">
-
-    ```bash
-    $ sudo add-apt-repository \
-       "deb [arch=amd64] $DOCKER_EE_URL/ubuntu \
-       $(lsb_release -cs) \
-       $DOCKER_EE_VERSION stable"
-       
-    ```
-
-    </div>
-    <div id="s390x_repo" class="tab-pane fade" markdown="1">
-
-    ```bash
-    $ sudo add-apt-repository \
-       "deb [arch=s390x] $DOCKER_EE_URL/ubuntu \
-       $(lsb_release -cs) \
-       $DOCKER_EE_VERSION stable"
-
-    ```
-
-    </div>
-    <div id="ppc64el_repo" class="tab-pane fade" markdown="1">
-
     ```bash
     $ sudo add-apt-repository \
-       "deb [arch=ppc64el] $DOCKER_EE_URL/ubuntu \
+       "deb [arch=$(dpkg --print-architecture)] $DOCKER_EE_URL/ubuntu \
```

Comment:
Does this print the correct architecture (i.e. `amd64`, not `x86_64`)?

Reply:
It does, that was the original instructions before we changed them. That's the how the instructions go in the get.docker.com script.